src/solace_ai_connector/common/utils.py

```diff
@@ -118,6 +118,7 @@ def import_module(name, base_path=None, component_package=None):
                     ".components.general",
                     ".components.general.for_testing",
                     ".components.general.langchain",
+                    ".components.general.openai",
```

Comment:
Did it take you a long time to find this? Sorry - somehow it was missed in my commit for the openai component

Reply:
No, the error said it couldn't fine/load the module, so I just looked where langchain was added, and duplicated that process